doc/src/SUMMARY.md

```diff
@@ -0,0 +1,23 @@
+# Summary
+
+[What is PL/Rust?](./plrust.md)
```

Comment:
To respond to your concern about whether this should be a question: I like the implied "question and answer" format here!